Formula/git-remote-gcrypt.rb

```diff
@@ -0,0 +1,19 @@
+class GitRemoteGcrypt < Formula
+  desc "git remote helper for GPG-encrypted remotes"
+  homepage "https://spwhitton.name/tech/code/git-remote-gcrypt/"
+  url "https://git.spwhitton.name/git-remote-gcrypt.git"
```

Comment:
@bfontaine sorry… I thought the `version 1.0.0` would do that (specify the tag from the repo). Refactored and verified.